blossom-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -1,5 +1,7 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
 com.blossomproject.autoconfigure.core.CommonAutoConfiguration,\
+com.blossomproject.autoconfigure.core.AsyncConfiguration,\
+com.blossomproject.autoconfigure.core.AsyncConfiguration.AsyncProperties,\
```

Comment:
Right, I was used to having the Properties class in its own file (instead of a nested class), so I had to put it in spring.factories for spring to find it